src/app/content/highlights/components/SummaryPopup/HighlightsHelpInfo.tsx

```diff
@@ -0,0 +1,96 @@
+import * as Cookies from 'js-cookie';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import styled, { css, keyframes } from 'styled-components';
+import { Times } from 'styled-icons/fa-solid/Times/Times';
+import { user as userSelector } from '../../../../auth/selectors';
+import { PlainButton } from '../../../../components/Button';
+import htmlMessage from '../../../../components/htmlMessage';
+import { linkStyle, textRegularStyle } from '../../../../components/Typography';
+import theme from '../../../../theme';
+
+// This is copied from CallToActionPopup > styles.tsx
+// Wher should we store this kind of functions?
+const slideInFromBottom = keyframes`
+  0% {
+    bottom: -100%;
+  }
+
+  100% {
+    bottom: 0;
+  }
+`;
+
+const slideInAnimation = css`
+  animation: ${800}ms ${slideInFromBottom} ease-out;
+`;
+
+// tslint:disable-next-line: variable-name
+const Wrapper = styled.div`
+  position: fixed;
+  bottom: 0;
+  z-index: ${theme.zIndex.highlightsHelpInfoMobile};
+  ${textRegularStyle}
```

Comment:
if you make this `bodyCopyRegularStyle` its the same thing but includes the anchor styling, you probably don't want the `text-decoration: none;`, which is the only thing you added that it doesn't have